docs/Product_Requirements_Document.md

```
@@ -0,0 +1,318 @@
+# Twiggle: Product Requirements Document (PRD)
+
+## Document Information
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)